src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -181,25 +181,7 @@ function IOURequestStepAmount({
             });
             const backendAmount = CurrencyUtils.convertToBackendAmount(Number.parseFloat(amount));
 
-            if (shouldSkipConfirmation) {
-                if (iouType === CONST.IOU.TYPE.SPLIT) {
-                    IOU.splitBill({
-                        participants,
-                        currentUserLogin: currentUserPersonalDetails.login ?? '',
-                        currentUserAccountID: currentUserPersonalDetails.accountID,
-                        amount: backendAmount,
-                        comment: '',
-                        currency,
-                        merchant: '',
-                        tag: '',
-                        category: '',
-                        created: transaction?.created ?? '',
-                        billable: false,
-                        iouRequestType: CONST.IOU.REQUEST_TYPE.MANUAL,
-                        existingSplitChatReportID: report?.reportID,
-                    });
-                    return;
-                }
+            if (shouldSkipConfirmation && iouType !== CONST.IOU.TYPE.SPLIT) {
```

Comment:
It doesn't make sense to skip the confirmation screen for the case of uneven splits. Because each split can be configured now. Though I think I should bring this back for Smartscan & Workspace splits.